version/1/4/sv/code_of_conduct.md

```diff
@@ -0,0 +1,75 @@
+# Medarbetarfördragets Uppförandekod
```

Comment:
I “*Contributor Covenant* Code of Conduct” så fungerar det som ett egennamn, och hur jag än gjorde kändes det tokigt, så jag hittade på ett svenskt egennamn.